packages/fuel-gauge/src/coverage-contract.test.ts

```diff
@@ -523,12 +512,12 @@ describe('Coverage Contract', () => {
       .echo_struct_vector_last([
         {
           foo: 1,
-          bar: 11337n,
+          bar: 11337,
```

Comment:
```suggestion
          bar: 11337n,
```

Why are we removing the BigInt expression here?

Reply:
Because we're getting compilation errors that `bigint` can't be passed as `BigNumberish`.